tests/integration/tasks/test-address-list.yml

```diff
@@ -1,39 +1,34 @@
 ---
-- name: ALWAYS_CHANGES Test adding a firewall address-list
-  mt_ip_firewall_addresslist:
+- name: Test adding a firewall address-list
```

Comment:
Why get rid of the ALWAYS_CHANGES string?  I think that helps verify the test output and the intention of the test.

Reply:
Because usually the first task in the test sets up the baseline to test. If this is the second or the third time the tests are run there is no guarantee that the list we are trying to ensure is different or missing.

Reply:
so we don't ensure any order?  only that the item with the specified parameters exists?

Reply:
What order?

Reply:
you said 

> Because usually the first task in the test sets up the baseline to test. If this is the second or the third time the tests are run there is no guarantee that the list we are trying to ensure is different or missing.

I took that to mean it's an ordering problem.  I don't understand why we cannot guarantee the idempotency of an address-list.

Reply:
In order to test something you need to establish a baseline against which you are testing. "Test adding a firewall address-list" is adding a baseline against which we can test adding or removing addresses from a list.

Reply:
I see what you're saying.  That task indeed shouldn't always change. 

Should we add another task after that says "NEVER_CHANGES" to test the idempotency?

Reply:
Oops, the third task should do that.

Reply:
ah ok... didn't catch that